ogr/ogrsf_frmts/sqlite/ogrsqlitebase.h

```diff
@@ -197,6 +197,7 @@ class OGRSQLiteBaseDataSource CPL_NON_FINAL : public GDALPamDataset
     GetLayerWithGetSpatialWhereByName(const char *pszName) = 0;
 
     virtual OGRErr AbortSQL() override;
+    bool SetQueryLoggerFunc(GDALQueryLoggerFunc pfnQueryLoggerFuncIn, void*poQueryLoggerArgIn) override;
```

Comment:
format linter not happy here

Reply:
yeah, I've fixed that but I'm stuck on ogr/ogrsf_frmts/sqlite/ogrsqlitedatasource.cpp:4695,style,unreachableCode,Statements following return, break, continue, goto or throw will never be executed.

Reply:
> yeah, I've fixed that but I'm stuck on ogr/ogrsf_frmts/sqlite/ogrsqlitedatasource.cpp:4695,style,unreachableCode,Statements following return, break, continue, goto or throw will never be executed.

See my suggestion of adding a #else / #endif that should hopefully fix it